modules/installation-aws-delete-cluster.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_aws/installing-aws-localzone.adoc
+
+:_content-type: PROCEDURE
+[id="installation-aws-delete-cluster"]
+= Deleting a cluster with a configured AWS Local Zone infrastructure
+
+After you install a cluster on Amazon Web Services (AWS) into an existing Virtual Private Cloud (VPC), and you set subnets for each Local Zone location, you can delete the cluster and any AWS resources associated with it. 
+
+The example in the procedure assumes that you created a VPC and its subnets by using a CloudFormation template.
+
+.Prerequisites
+
+* You know the name of the CloudFormation stack, such as `cluster-vpc`. You need the name of the stack to delete the cluster.
+* You can access rights to the directory that contains the installation files that were created by the installation program. 
+* Your account includes a policy that provides you with permissions to delete the CloudFormation stack.
+
+.Procedure
+
+. Change to the directory that contains the stored installation files, and delete the cluster by using the `destroy` command:
++
+[source,terminal]
+----
+$ ./openshift-install destroy cluster --dir <installation_directory> \//<1> 
+   --log-level=debug <2>
+----
+<1> For `<installation_directory>`, specify the location where the installation program stored your installation files.
```

Comment:
```suggestion
. Change to the directory that contains the installation program, and delete the cluster by using the `destroy cluster` command:
+
[source,terminal]
----
$ ./openshift-install destroy cluster --dir <installation_directory> \//<1> 
   --log-level=debug <2>
----
<1> For `<installation_directory>`, specify the location where the installation files are stored.
```

Reply:
Updated, but what creates the installation files? The installation program? I would usually need to state what entity does an action.